changelogs/CHANGELOG-v0.3.0.md

```diff
@@ -0,0 +1,38 @@
+The v0.3.0 release is our second official minor release of the the bpfd project :tada
+
+The following describes some of the major new features/updates: 
+
+- The ability to list and get kernel information for ALL programs regardless
+of if they were loaded by bpfd or another process
+
+- Deprecation of bpfd specific UUIDs for each loaded program in favor of the
+standard generated kernel ID for all programs regardless of what process loaded them.
+
+- Support for some new bpf program types: 
+    * `Uprobe`s
+    * `Uretprobe`s
+    * `Kprobes`s
+    * `KretProbes`s
```

Comment:
I think the `s` should be dropped on probe types above. The generated markdown looks weird.